_notebooks/2020-12-12-Get-and-Extract-Data.ipynb

```diff
@@ -24,7 +24,8 @@
     "# hide\n",
```

Comment:
<p>Why is this <code>line_margin=0.7</code>  ? What file/page did we try this out on?</p><p> </p><p>Can we please use verbose names instead of <code>rsrcmgr</code> ? </p><p> </p><p>What do we need <code>StringIO()</code> for? Can we use native <code>str</code> instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/projectnoor/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Used line_margin=0.7 because anything below that was considering a paragraph break as a different text blob(bounding box?)</p>